crazyclock.ino

```diff
@@ -79,18 +77,23 @@ void setup() {
   bool wifiAvailable = isWiFiAvailable(&lcd, ssid, password);
   if (wifiAvailable) {
 
-    bool isNtpAvailable = retrieveEpochTimeFromNTP(lcd, &epochSeconds);
+    timeClient.begin();
+    bool isNtpAvailable = tryNTPTimeClientUpdate(timeClient);
     if (isNtpAvailable) {
 
-      Serial.println("Updating RTC with time from NTP...");
-      LocalDateTime localDateTime = plDateTimeConverter.fromUtc(epochSeconds);
-      int mH = localDateTime.getLocalTimeFragment(HOURS);
-      int mM = localDateTime.getLocalTimeFragment(MINUTES);
-      int mS = localDateTime.getLocalTimeFragment(SECONDS);
-      rtc.setHour(mH);
-      rtc.setMinute(mM);
-      rtc.setSecond(mS);
+      Serial.println("Updating RTC with UTC time from NTP...");
+      if (timeClient.update() && timeClient.isTimeSet()) {
+        epochSeconds = timeClient.getEpochTime();
+        rtc.setEpoch(epochSeconds);
```

Comment:
RTC clock is not aware of timezones. So it is not enough to calculate local (Warsaw timezone) time once.

We will need to convert it every time: from real time to local fake time in Warsaw timezone.